x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.test.ts

```diff
@@ -140,7 +140,9 @@ const baseRuleParams: Pick<RuleParams, 'count' | 'timeSize' | 'timeUnit' | 'logV
 
 const TIMESTAMP_FIELD = '@timestamp';
 const FILEBEAT_INDEX = 'filebeat-*';
-const EXECUTION_TIMESTAMP = new Date('2022-01-01T00:00:00.000Z').valueOf();
+const EXECUTION_TIMERANGE = {
+  lte: new Date('2022-01-01T00:00:00.000Z').valueOf(),
```

Comment:
Since we might have `gte` shall we add a test for that case too?
```
  const to = executionTimeRange?.lte || Date.now();
  const from = executionTimeRange?.gte || to - intervalAsMs;
```

Reply:
@maryam-saeidi, no need here. The executor doesn't use `gte`